samples/msal-node-samples/ElectronReactTestApp/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "electron-react-test-app",
+  "productName": "electron-react-test-app",
+  "version": "1.0.0",
+  "description": "A sample Electron and React application that uses msal-node to acquire tokens",
+  "main": ".webpack/main",
+  "scripts": {
+    "build": "tsc",
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "build:package": "cd ../../../lib/msal-common && npm run build && cd ../msal-node && npm run build",
+    "start:build": "npm run build:package && npm start",
+    "lint": "eslint --ext .ts,.tsx ."
+  },
+  "devDependencies": {
+    "@electron-forge/cli": "^6.0.0-beta.63",
+    "@electron-forge/maker-deb": "^6.0.0-beta.63",
+    "@electron-forge/maker-rpm": "^6.0.0-beta.63",
+    "@electron-forge/maker-squirrel": "^6.0.0-beta.63",
+    "@electron-forge/maker-zip": "^6.0.0-beta.63",
+    "@electron-forge/plugin-webpack": "^6.0.0-beta.63",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.0",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-redux": "^7.1.22",
+    "@typescript-eslint/eslint-plugin": "^5.9.1",
+    "@typescript-eslint/parser": "^5.9.1",
+    "@vercel/webpack-asset-relocator-loader": "^1.7.0",
+    "css-loader": "^6.5.1",
+    "electron": "16.0.7",
+    "eslint": "^8.6.0",
+    "eslint-plugin-import": "^2.25.4",
+    "fork-ts-checker-webpack-plugin": "^6.5.0",
+    "node-loader": "^2.0.0",
+    "style-loader": "^3.3.1",
+    "ts-loader": "^9.2.6",
+    "typescript": "~4.5.4"
+  },
+  "dependencies": {
+    "@azure/msal-node": "^1.8.0",
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@google-cloud/speech": "^4.9.0",
```

Comment:
there are a lot of dependencies here that we should remove as they are either not used or unnecessary: eslint, redux etc.

Reply:
Done!